src/contributing/void-docs/style-guide.md

```diff
@@ -63,27 +63,38 @@ For example:
 # vi /etc/fstab
 ```
 
-And not:
+and not:
 
 ```
 $ sudo vi /etc/fstab
 ```
 
-And also not:
+and also not:
 
 ```
 vi /etc/fstab
 ```
 
 ## Links
 
+Link text should not include sentence-level punctuation. For example:
+
+```
+[Visit this site](https://example.org).
+```
+
+and not:
+
+```
+[Visit this site.](https://example.org)
+```
+
 ### Internal links
 
 Links to other sections of the Handbook must be relative. For example:
 
 ```
 [example](./example.html#heading)
```

Comment:
(from your last style-guide PR) should this be example.md#heading ? Most current links point to .md files

Reply:
i wasn't sure whether, when referring to anchors in links, we need to specify '.html' or '.md' in the source; istr some places in the doc sources specifying '.html'. If `./example.md#heading` definitely works, yes, we should certainly change it (and i'll do so in this PR).

Reply:
yes, you can find an internal link to .md with a fragment in src/config/microcode.md.

Reply:
@jeffayle: i've just tried building with links that use '.md' instead of '.html', and that seems to work, so i've updated this PR accordingly. Thanks for reminding me to check this!